docs/SMCWG-charter.md

```diff
@@ -0,0 +1,328 @@
+be out of scope.
+
+### 3. Charter Expiration
+
+The SMCWG is chartered indefinitely until it is dissolved as specified in Bylaw
+5.3.2(c).
+
+### 4. Personnel and Participation
+
+#### 4.1. Selection of Officers
+
+Stephen Davidson will act as chair of the SMCWG until the first Working Group
+Teleconference, at which time the group will elect a chair and vice-chair. The
+chair and vice-chair will serve until October 31, 2022, or until they are
+replaced, resign, or are otherwise disqualified. Thereafter, elections SHALL be
+held for chair and vice chair every two years in coordination with the Forum’s
+election process and in conjunction with its election cycle. Voting SHALL occur
+in accordance with Bylaw 4.1(c).
+
+#### 4.2. Participation
+
+##### 4.2.1. Eligibility to Participate
+
+The SMCWG SHALL consist of two classes of voting members, Certificate Issuers
+and Certificate Consumers meeting the eligibility criteria below.
+
+1. A Certificate Issuer eligible for voting membership in the SMCWG MUST have
+   a publicly-available audit report or attestation statement in accordance
+   with one of the following schemes:
+
+    * WebTrust for CAs v.2.0 or newer; or
+    * ETSI EN 319 411-1, which includes normative references to ETSI EN 319 401
+      (the latest version of the referenced ETSI documents should be applied).
+
+   These audit reports must also meet the following requirements:
+
+    * They must report on the operational effectiveness of controls for a
+      historic period of at least 60 days;
+    * No more than 27 months have elapsed since the beginning of the
+      reported-on period and no more than 15 months since the end of the
+      reported-on period; and
+    * The audit report was prepared by a Qualified Auditor.
+
+   In addition, the Certificate Issuer MUST actively issue S/MIME certificates
+   that are treated as valid by at least one Certificate Consumer that produces
+   a mail user agent or email service provider that processes S/MIME
+   certificates.
+
+2. A Certificate Consumer eligible for voting membership in the SMCWG must
+   produce and maintain a mail user agent (web-based or application based) or
+   email service provider that processes S/MIME certificates.
+
+The SMCWG SHALL allow participation by Interested Parties, as set
+forth in the Bylaws.
+
+##### 4.2.2. Membership Application/Declaration Process
+
+1. An Applicant not already a member of the Forum SHALL provide the following
+   information:
+
+   * Confirmation that the applicant satisfies at least one (1) of the
+     membership eligibility criteria (and if it satisfies more than one (1),
+     indication of the single category under which the applicant wishes to apply).
+   * The organization name, as they wish it to appear on the Forum Web site and
+     in official Forum documents.
+   * URL of the applicant’s main Web site.
+   * Names and email addresses of employees who will participate in the Working
```

Comment:
```suggestion
   * Names and email addresses of designated Representatives who will participate in the Working
```

This tries to tackle the employment issue, especially in situations where a policy management authority may contract to a third-party operator.

Reply:
@wthayer Curious for any comments you might have as well

Reply:
SGTM

Reply:
Are there (should there be?) any restrictions around who can be a designated Representative?

Reply:
No, people should be free to be able to choose their own Representatives.